client/src/components/History.tsx

```diff
@@ -46,6 +46,17 @@ const Entry = React.memo(
   }) => {
     const path =
       value.request.path + formatQueryParams(value.request.query_params);
+
+    let responseStatusColor = "blue";
+    if (value.response.status >= 600) {
+      responseStatusColor = "magenta";
+    } else if (value.context.mock_type === "proxy") {
+      if (value.response.status >= 500) {
+        responseStatusColor = "red";
+      } else if (value.response.status >= 400) {
```

Comment:
je mettrais pas de couleur pour les 400, c'est des comportements normaux

Reply:
Pas forcément dans le cas d'un proxy imagine t'avais des tests qui passaient mais t'as fais des modifs côté services genre t'as rajouté de la validation et du coup to call passe en 400 en soit ton test et pété et la 400 fait partie des choses que t'as envie de pv grep rapidement